packages/react-native/React/Base/RCTConvert.m

```diff
@@ -900,7 +913,15 @@ + (UIColor *)UIColor:(id)json
   } else if ([json isKindOfClass:[NSDictionary class]]) {
     NSDictionary *dictionary = json;
     id value = nil;
-    if ((value = [dictionary objectForKey:@"semantic"])) {
+    if ((value = [dictionary objectForKey:@"display-p3"]) ||
+        (value = [dictionary objectForKey:@"srgb"])) {
```

Comment:
if we change the js representation, this could be something like
```objc
NSString * colorSpace = [dictionary objectForKey: @"space"];
if ([@[@"display-p3", @"srgb"] containsObject:colorSpace]) {
  RCTColorSpace space = [RCTColorSpace fromString:colorSpace];
	//...
}